core/src/main/java/org/mule/runtime/core/exception/ErrorTypeRepositoryFactory.java

```diff
@@ -8,6 +8,7 @@
 
 import static org.mule.runtime.core.exception.Errors.ComponentIdentifiers.CONNECTIVITY;
 import static org.mule.runtime.core.exception.Errors.ComponentIdentifiers.EXPRESSION;
+import static org.mule.runtime.core.exception.Errors.ComponentIdentifiers.FILTERED;
```

Comment:
Again, FILTERED -> Either something reused from validations or DUPLICATE_MESSAGE if we need something specific.